src/components/CytoscapeGraph/graphs/GraphStyles.ts

```diff
@@ -361,8 +385,24 @@ export class GraphStyles {
       return `<div class="${labelDefault} ${labelBox}" style="${labelStyle}">${icons}${contentSpan}</div>`;
     }
 
-    const contentSpan = `<div class="${contentClasses}" style="${contentStyle}">${contentText}</div>`;
-    return `<div class="${labelDefault}" style="${labelStyle}">${icons}${contentSpan}</div>`;
+    let hosts: string[] = [];
+    node.hasVS?.hostnames?.forEach(h => hosts.push(h === '*' ? '(All hosts)' : h));
```

Comment:
I'm not sure, maybe just showing `*` would be better than `(All Hosts)`.  I actually found `(All Hosts)` a little confusing.  Or, perhaps both `* (all hosts)`